core/CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 
 [1]: https://pypi.org/project/google-cloud-core/#history
 
+## 0.29.0
+
+12-03-2018 15:54 PST
+
+### Breaking Changes
+- Remove iam module from core. ([#6775](https://github.com/googleapis/google-cloud-python/pull/6775))
```

Comment:
As a breaking change, this needs a note telling users how to fix things, i.e. that it is now importable from `google.api_core.iam`.